commands/microservice

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+export LC_ALL=C.UTF-8
+help() {
+  cat <<EOF
+
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+    Usage: c8y bpl $(basename "$0") <commands> --id <microservice-id>
+    Commands:
+      subscribe           Subscribe tenant to microservice
+      unsubscribe         Unsubscribe tenant to microservice
+      help                Show this help message and exit
+    Arguments:
+      -i, --id      ID    The ID of the application/microservice (required)
+      -h, --help          Show this help message and exit
```

Comment:
in general: 
- it would be nice if the standard parameters like --dry/--debug could be used; but I guess that would require some work from Reuben; maybe you can talk to him about that
- the same applies to autocompletion, e.g. for the names of the microservices